src/js/services/sectionShareButton.js

```diff
@@ -0,0 +1,109 @@
+import { createPopper } from '@popperjs/core';
+
+export const init = () => {
+  const showEvents = ['mouseover', 'hover', 'mouseenter', 'focus'];
+  const hideEvents = ['mouseout', 'mouseleave', 'blur'];
+  const clickEvents = ['click'];
+  const tooltip = document.querySelector('.tooltip-popover');
+
+  // Give article headings direct links to anchors
+  $('article h1, article h2, article h3, article h4, article h5, article h6')
+    .filter('[id]')
+    .each(function () {
+      var isMainTitle = $(this).prop('nodeName') === 'H1';
+      $(this).append(
+        (isMainTitle ? ' <a href="#' : '<a href="#' + $(this).attr('id')) +
+          '"><i class="fa fa-link"></i></a>',
+      );
+      if (isMainTitle) {
+        $(this).find('i').toggle();
+      }
+    });
+
+  const makePopper = (icon) =>
+    Object.assign(icon, {
+      show() {
+        tooltip.setAttribute('data-show', '');
+        // change tooltip text based on current button popover.
+        tooltip.innerHTML = "Copy link<div id='arrow' data-popper-arrow></div>";
+        icon.instance = createPopper(icon, tooltip, {
+          modifiers: [
+            {
+              name: 'offset',
+              options: {
+                offset: [0, 8],
+              },
+            },
+          ],
+        });
+        window.AnalyticsClient.trackAction('docs-share-button-hover', {
+          page: location.pathname,
+          success: true,
+        });
```

Comment:
I think we can remove that event now. I don't thing it gives us much insight now that we are out of the experimentation phase

Reply:
Good call. I was on the fence about the hover one too but I can remove it 👍